geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Monitoring.scala

```diff
@@ -0,0 +1,30 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.utils.geotools
+
+import com.google.gson.{Gson, GsonBuilder}
+import com.typesafe.scalalogging.LazyLogging
+
+object Monitoring extends LazyLogging {
```

Comment:
for configuring the logger in e.g. log4j.xml, are we going to have to use `Monitoring$` or something? maybe we should explicitly create the logger so we get a better name.